### ISSUE TITLE
Unexpected token when running kappa through docker-compose

### ISSUE THREAD

**Issue by goloroden** [krakenjs/kappa#119]
### Steps to reproduce

- Clone the `kappa` repository
- Run `docker-compose up` (in my case the IP of VMware is `192.168.9.130`)
- Create a demo directory, e.g. `~/foo`
- Switch to that directory, i.e. `cd ~/foo`
- Try to install express using kappa: `npm install express --registry http://192.168.9.130:8000`

### Expected result

- kappa does not find express in the local registry
- kappa fetches express from the official npm registry

### Actual result

- kappa does not find express in the local registry
- kappa throws a few exceptions (the details of the exceptions are shown below)

#### Details of the exceptions

```
[…]
registry_1 | [error] [<0.105.0>] OS Process Error <0.2197.0> :: {not_found,
registry_1 |                                                     <<"document not found">>}
registry_1 | [info] [<0.105.0>] 172.18.0.3 - - GET /registry/_design/app/_show/package/express?pkg=express 404
kappa_1    | 160525/073826.927, internalError, message: Unexpected token  stack: SyntaxError: Unexpected token 
kappa_1    |     at Object.parse (native)
kappa_1    |     at Object.internals.tryParseBuffer (/usr/lib/node_modules/kappa/node_modules/wreck/lib/index.js:467:25)
kappa_1    |     at finish (/usr/lib/node_modules/kappa/node_modules/wreck/lib/index.js:308:28)
kappa_1    |     at wrapped (/usr/lib/node_modules/kappa/node_modules/hoek/lib/index.js:866:20)
kappa_1    |     at onReaderFinish (/usr/lib/node_modules/kappa/node_modules/wreck/lib/index.js:346:16)
kappa_1    |     at g (events.js:260:16)
kappa_1    |     at emitNone (events.js:72:20)
kappa_1    |     at emit (events.js:166:7)
kappa_1    |     at finishMaybe (_stream_writable.js:468:14)
kappa_1    |     at endWritable (_stream_writable.js:478:3)
kappa_1    | 160525/073826.927, request, http://0.0.0.0:8000: get /express {} 500 (1174ms) 
kappa_1    | 160525/073837.594, ops, memory: 34Mb, uptime (seconds): 1906.072, load: 0.0029296875,0.0146484375,0.04541015625
registry_1 | [error] [<0.2187.0>] OS Process Error <0.2197.0> :: {not_found,
registry_1 |                                                      <<"document not found">>}
registry_1 | [info] [<0.2187.0>] 172.18.0.3 - - GET /registry/_design/app/_show/package/express?pkg=express 404
kappa_1    | 160525/073838.115, internalError, message: Unexpected token  stack: SyntaxError: Unexpected token 
kappa_1    |     at Object.parse (native)
kappa_1    |     at Object.internals.tryParseBuffer (/usr/lib/node_modules/kappa/node_modules/wreck/lib/index.js:467:25)
kappa_1    |     at finish (/usr/lib/node_modules/kappa/node_modules/wreck/lib/index.js:308:28)
kappa_1    |     at wrapped (/usr/lib/node_modules/kappa/node_modules/hoek/lib/index.js:866:20)
kappa_1    |     at onReaderFinish (/usr/lib/node_modules/kappa/node_modules/wreck/lib/index.js:346:16)
kappa_1    |     at g (events.js:260:16)
kappa_1    |     at emitNone (events.js:72:20)
kappa_1    |     at emit (events.js:166:7)
kappa_1    |     at finishMaybe (_stream_writable.js:468:14)
kappa_1    |     at endWritable (_stream_writable.js:478:3)
kappa_1    | 160525/073838.115, request, http://0.0.0.0:8000: get /express {} 500 (766ms) 
kappa_1    | 160525/073852.599, ops, memory: 35Mb, uptime (seconds): 1921.078, load: 0.0029296875,0.0146484375,0.04541015625
kappa_1    | 160525/073907.605, ops, memory: 35Mb, uptime (seconds): 1936.083, load: 0.0029296875,0.0146484375,0.04541015625
kappa_1    | 160525/073922.607, ops, memory: 35Mb, uptime (seconds): 1951.084, load: 0.0029296875,0.0146484375,0.04541015625
kappa_1    | 160525/073937.615, ops, memory: 34Mb, uptime (seconds): 1966.093, load: 0.0029296875,0.0146484375,0.04541015625
registry_1 | [error] [<0.106.0>] OS Process Error <0.2197.0> :: {not_found,
registry_1 |                                                     <<"document not found">>}
registry_1 | [info] [<0.106.0>] 172.18.0.3 - - GET /registry/_design/app/_show/package/express?pkg=express 404
kappa_1    | 160525/073938.888, internalError, message: Unexpected token  stack: SyntaxError: Unexpected token 
kappa_1    |     at Object.parse (native)
kappa_1    |     at Object.internals.tryParseBuffer (/usr/lib/node_modules/kappa/node_modules/wreck/lib/index.js:467:25)
kappa_1    |     at finish (/usr/lib/node_modules/kappa/node_modules/wreck/lib/index.js:308:28)
kappa_1    |     at wrapped (/usr/lib/node_modules/kappa/node_modules/hoek/lib/index.js:866:20)
kappa_1    |     at onReaderFinish (/usr/lib/node_modules/kappa/node_modules/wreck/lib/index.js:346:16)
kappa_1    |     at g (events.js:260:16)
kappa_1    |     at emitNone (events.js:72:20)
kappa_1    |     at emit (events.js:166:7)
kappa_1    |     at finishMaybe (_stream_writable.js:468:14)
kappa_1    |     at endWritable (_stream_writable.js:478:3)
kappa_1    | 160525/073938.888, request, http://0.0.0.0:8000: get /express {} 500 (848ms) 
[…]
```

**Comment by jasisk** [comment]
You're right—I need to update the docker-compose file to use the new version. Thanks for the reminder.

**Comment by shaunwarman** [comment]
Fixed via #122
